### ISSUE TITLE
Clarity: Enchantments

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#482]
If an item has 10% chance to trigger, and it gets a luck enchantment (+10%), will it then get to 11%, 19% or 20% chance to trigger?

That is, is it additive, multiplicative in series, or a direct modifier?

The same question for impact and grace.
Especially impact, since there's a radical difference for e.g. charged strikes, AoE heals and HoTs:
Mighty blow: 1000% -> +100% or +10%
AoE Heal: 15% -> +1.5% or +10%

Unless I missed it, I don't see any way to figure this out except by looking at the code or doing a lot of testing. So perhaps there should be some clarity in the description.

**Comment by ImperialUser** [comment]
It increases the base amount.  So 10% on 10% would be 11%.

**Comment by askebv** [comment]
Right, I got confirmation from Travis on this. It still could be clearer from the tooltips though.
